src/Features/DistantTreeLighting.cpp

```diff
@@ -48,10 +49,11 @@ void DistantTreeLighting::DrawSettings()
 		if (ImGui::IsItemHovered()) {
 			ImGui::BeginTooltip();
 			ImGui::PushTextWrapPos(ImGui::GetFontSize() * 35.0f);
-			ImGui::Text("Subsurface Scattering (SSS) amount\n");
-			ImGui::Text("Soft lighting controls how evenly lit an object is.\n");
-			ImGui::Text("Back lighting illuminates the back face of an object.\n");
-			ImGui::Text("Combined to model the transport of light through the surface.");
+			ImGui::Text(
+				"Subsurface Scattering (SSS) amount "
```

Comment:
Add a . to the end

Reply:
I've made these changes, just need to figure out why I can't push my branch to origin now:
![image](https://github.com/doodlum/skyrim-community-shaders/assets/78163326/aa57ed5c-0cc6-4bb3-957c-a20b3d9fcba2)
I think it might have to do with the tag created when I ran the Build workflow. I'll figure it out shortly

Reply:
Ok I was able to push my changes. I think doing a git pull on my fork also pulled the tags associated with each release, and git can't differentiate between the tag and the branch with the above command.
I ended up deleting the local tag following https://stackoverflow.com/a/44299883 and I could push my changes.

Reply:
Fixed in 358ac33f14939cdc2eb5b4a2a33b9820ee888e2b